conf/core/ConfigOptions.asciidoc

```diff
@@ -211,6 +211,15 @@ Invalid relation members will cause a warning to be logged.  You should enable t
 are loading data that has not been previously validated in another OSM API database.  Enabling this
 setting may cause writing to an OSM API database to occur more slowly.
 
+=== apidb.reader.tag.immediately.connected.out.of.bounds.ways
+
+* Data Type: bool
+* Default Value: `false`
+
+If enabled, all ways that are outside of the bounds specified by convert.bounding.box when
+performing a bounded API database query but are immediately connected to a way that falls within or
+crosses the bounds are tagged with a custom tag.
+
 === arff.writer.precision
```

Comment:
This option helps prevent replacement changesets from dropping nodes in ways that are completely outside of the query AOI but are connected to a way crossing the AOI bounds.